scripts/globals/mobskills.lua

```diff
@@ -288,15 +288,23 @@ xi.mobskills.mobMagicalMove = function(mob, target, skill, damage, element, dmgm
     local finaldmg = damage * mab * dmgmod
 
     -- get resistance
-    local avatarAccBonus = 0
+    local petAccBonus = 0
     if mob:isPet() and mob:getMaster() ~= nil then
         local master = mob:getMaster()
         if mob:isAvatar() then
-            avatarAccBonus = utils.clamp(master:getSkillLevel(xi.skill.SUMMONING_MAGIC) - master:getMaxSkillLevel(mob:getMainLvl(), xi.job.SMN, xi.skill.SUMMONING_MAGIC), 0, 200)
+            petAccBonus = utils.clamp(master:getSkillLevel(xi.skill.SUMMONING_MAGIC) - master:getMaxSkillLevel(mob:getMainLvl(), xi.job.SMN, xi.skill.SUMMONING_MAGIC), 0, 200)
+        end
+
+        local skillchainTier, _ = xi.magicburst.formMagicBurst(element, target)
+        if
+            mob:getPetID() > 0 and
+            skillchainTier > 0
+        then
+            petAccBonus = petAccBonus + 25
```

Comment:
Where does this 25 bonus macc come from? Any source?

Reply:
Pulled this from an xi.magic function. I assumed it was a global bonus during a magic burst

Reply:
yeah Jimmy seems to think it should be 30? he's looking into it so standby

Reply:
I've updated to +30, lmk if it ends up being something else

Reply:
Looks like it's 25, there's some ancient blog that says that

Reply:
snip snap. Updated back to +25. Thanks for verifying